cassandra-plugins/src/main/java/co/cask/plugin/etl/batch/source/BatchCassandraSource.java

```diff
@@ -92,7 +93,8 @@ public void prepareRun(BatchSourceContext context) throws Exception {
 
     if (!Strings.isNullOrEmpty(config.properties)) {
       for (String pair : config.properties.split(",")) {
-        conf.set(pair.split(":")[0], pair.split(":")[1]);
+        conf.set(CharMatcher.WHITESPACE.trimFrom(pair.split(":")[0]),
```

Comment:
I am removing only the leading and trailing here as I assume the value of some key might have spaces.